src/ray/raylet/local_task_manager.h

```diff
@@ -54,7 +55,7 @@ namespace raylet {
 /// as it should return the request to the distributed scheduler if
 /// resource accusition failed, or a task has arguments pending resolution for too long
 /// time.
-class LocalTaskManager {
+class LocalTaskManager : public ILocalTaskManager {
```

Comment:
We currently have two naming convention for interface `ILocalTaskManager` and `LocalTaskManagerInterface`, lets converge to one? (Either is fine to me)